app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import LabPanelFieldSet from './LabPanelFieldSet';
 import LabTestFieldSet from './LabTestFieldSet';
+import ActiveOrders from './ActiveOrders';
+import PastOrders from './PastOrders';
+import Accordion from '../accordion';
```

Comment:
Is accordion supposed to be small a or capital A?

Reply:
The accordion there is referring to a folder and not a file, the code is exported from an index.js file inside the accordion directory.